src/py/flwr/client/app.py

```diff
@@ -491,14 +492,11 @@ def _load_client_app() -> ClientApp:
                     # Load ClientApp instance
                     client_app: ClientApp = load_client_app_fn()
 
+                    # Execute ClientApp
                     reply_message = client_app(message=message, context=context)
-                    # Update node state
-                    node_state.update_context(
-                        run_id=message.metadata.run_id,
-                        context=context,
-                    )
+
                 except Exception as ex:  # pylint: disable=broad-exception-caught
-                    log(ERROR, "ClientApp raised an exception", exc_info=ex)
+                    log(ERROR, "SuperNode raised an exception", exc_info=ex)
```

Comment:
Exception is coming from the `ClientApp`, no?
```suggestion
                    log(ERROR, "ClientApp raised an exception", exc_info=ex)
```